### ISSUE TITLE
The role failed to download arm release because it is searching aarch64

### ISSUE THREAD

**Issue by billmetangmo** [darkwizard242/ansible-role-lazydocker#6]
Hello @darkwizard242 

Thanks for sharing this role with the community.

##### SUMMARY
Ansible role is not able to install lazydocker in arm because it tried to get it from repo using aarch64 with error:

![image](https://user-images.githubusercontent.com/25366207/198828115-f1fa265b-e67a-4d90-b843-0098e23df159.png)

##### ISSUE TYPE
- Bug Report

##### ANSIBLE VERSION

```paste below
ansible [core 2.11.12] 
  config file = None
  configured module search path = ['/home/bafou/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/bafou/.local/lib/python3.8/site-packages/ansible
  ansible collection location = /home/bafou/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/bafou/.local/bin/ansible
  python version = 3.8.10 (default, Jun 22 2022, 20:18:18) [GCC 9.4.0]
  jinja version = 3.1.2
  libyaml = True
```

##### CONFIGURATION

```paste below
```

##### OS / ENVIRONMENT
RAS

##### STEPS TO REPRODUCE

Execute the following role to install lazydocker on an arm instance
```yaml
- name: Install lazydocker
  ansible.builtin.import_role:
    name: lazydocker
```

##### EXPECTED RESULTS

Lazydocker is installed on the machine

##### ACTUAL RESULTS
Lazydocker is not installed , the playbook failed with error
```paste below
TASK [lazydocker : Debian/Ubuntu Family | Downloading and extracting lazydocker 0.19.0] ************************************************************************************************
fatal: [XXXXXXX]: FAILED! => changed=false 
  msg: 'Failure downloading https://github.com/jesseduffield/lazydocker/releases/download/v0.19.0/lazydocker_0.19.0_Linux_aarch64.tar.gz, HTTP Error 404: Not Found'

```

##### SCREENSHOTS

Regards

**Comment by billmetangmo** [comment]
This seems to be because for aarch64 instance, you need to download the arm64 ones as referred in the install_script:

```bash
#!/bin/bash

# allow specifying different destination directory
DIR="${DIR:-"$HOME/.local/bin"}"

# map different architecture variations to the available binaries
ARCH=$(uname -m)
case $ARCH in
    i386|i686) ARCH=x86 ;;
    armv6*) ARCH=armv6 ;;
    armv7*) ARCH=armv7 ;;
    aarch64*) ARCH=arm64 ;;
esac

# prepare the download URL
GITHUB_LATEST_VERSION=$(curl -L -s -H 'Accept: application/json' https://github.com/jesseduffield/lazydocker/releases/latest | sed -e 's/.*"tag_name":"\([^"]*\)".*/\1/')
GITHUB_FILE="lazydocker_${GITHUB_LATEST_VERSION//v/}_$(uname -s)_${ARCH}.tar.gz"
GITHUB_URL="https://github.com/jesseduffield/lazydocker/releases/download/${GITHUB_LATEST_VERSION}/${GITHUB_FILE}"

# install/update the local binary
curl -L -o lazydocker.tar.gz $GITHUB_URL
tar xzvf lazydocker.tar.gz lazydocker
install -Dm 755 lazydocker -t "$DIR"
rm lazydocker lazydocker.tar.gz
```
cf https://github.com/jesseduffield/lazydocker/blob/master/scripts/install_update_linux.sh

As hotfix, for my usecase , i set lazydocker_dl_url to arm64:

```yaml
- name: Install lazydocker
  ansible.builtin.import_role:
    name: lazydocker
  vars:
    lazydocker_dl_url: "https://github.com/jesseduffield/lazydocker/releases/download/v0.19.0/lazydocker_0.19.0_Linux_arm64.tar.gz"
```

**Comment by darkwizard242** [comment]
Hi @billmetangmo  - Thanks you for identifying the issue and bringing it to my attention. I will try to get a fix in by this week. Additionally, open to the idea of a PR with fix from your side. Otherwise, will get a fix in as time permits this week.

**Comment by darkwizard242** [comment]
Hi @MozkaGit  - Thanks very much for volunteering to improve the role and identifying issues.

I have a release [2.6.0](https://github.com/darkwizard242/ansible-role-lazydocker/releases/tag/2.6.0), that looks up ansible_architecture and assigns an appropriate value based on that tarballs that lazydocker releases have. 

Changes were merged in PR #16 , feel free to give that a look and utilize latest/2.6.0 version of the role on an aarch64 system. If it works, kindly let me know and we can close this bugfix and the other PR #15

**Comment by darkwizard242** [comment]
Hi @billmetangmo  - I have released [2.6.0](https://github.com/darkwizard242/ansible-role-lazydocker/releases/tag/2.6.0), that looks up ansible_architecture and assigns an appropriate value based on that tarballs that lazydocker releases have. 

Changes were merged in PR https://github.com/darkwizard242/ansible-role-lazydocker/pull/16 , feel free to give that a look and utilize latest/2.6.0 version of the role on an aarch64 system.

Closing the issue as another contributor has also verified it.
